### PR TITLE
Browser Tab support

### PR DESCRIPTION
**Pull request by theredsix**
Added ability for python version to switch between browser tabs while accomplishing tasks

### CODE REVIEW

File: python/src/cerebellum/browser.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 from abc import ABC, abstractmethod
 from dataclasses import dataclass
 from enum import StrEnum
-from typing import Any, Literal
+from typing import Any, Dict, List, Literal
```
Comment (scm-aiml) [inline python/src/cerebellum/browser.py:12]: @theredsix for maintained a typed library, I was pushing for the built-ins that can be leveraged for types, particularly items like `Dict` -> `dict` and `List` -> `list`.

Comment (theredsix) [reply python/src/cerebellum/browser.py:12]: I used the older types for backward compatibility with python 3.8. We can switch to lowercase if we bump up the minimum supported language version.

Comment (scm-aiml) [reply python/src/cerebellum/browser.py:12]: I would recommend we bump minimum version to `3.9` as `3.8` officially reached EOL in October this year.

File: python/src/cerebellum/planners/anthropic.py [hunk 2]
```diff
@@ -481,7 +495,7 @@ def format_into_messages(
         messages.extend([msg0, msg1])
 
         for past_step_idx, past_step in enumerate(session_history):
-            options = MsgOptions(mouse_position=False, screenshot=False, url=True)
+            options = MsgOptions(mouse_position=False, screenshot=False, tabs=False)
 
             if past_step_idx <= (len(session_history) - self.screenshot_history):
```
Comment (scm-aiml) [inline python/src/cerebellum/planners/anthropic.py:500]: it looks like `MsgOptions` no longer has a `url` attribute. I don't believe that attribute is expected anywhere so this line may be redundant.

File: python/src/cerebellum/planners/anthropic.py [hunk 3]
```diff
@@ -481,7 +495,7 @@ def format_into_messages(
...
                 options.url = True  # Matches TypeScript: options.url = true
```
Comment (scm-aiml) [inline python/src/cerebellum/planners/anthropic.py:501]: it looks like `MsgOptions` no longer has a `url` attribute. I don't believe that attribute is expected anywhere so this line may be redundant.

**Comment by scm-aiml** [comment]
@theredsix I pushed a set of updates to fix some type compatibility errors and to convert action types to an enum.

File: python/src/cerebellum/browser.py (hunk 1 above)
Comment (theredsix) [reply python/src/cerebellum/browser.py]: Agreed, set to 3.9

File: python/src/cerebellum/planners/anthropic.py (hunk 3 above)
Comment (theredsix) [reply python/src/cerebellum/planners/anthropic.py]: Good catch

**Pull request merged**